mesh/pncmesh.hpp

```diff
@@ -67,7 +67,7 @@ class ParNCMesh : public NCMesh
    /// Construct by partitioning a serial NCMesh.
    /** SFC partitioning is used by default. A user-specified partition can be
        passed in 'part', where part[i] is the desired MPI rank for element i. */
-   ParNCMesh(MPI_Comm comm, const NCMesh& ncmesh, int* part = NULL);
+   ParNCMesh(MPI_Comm comm, const NCMesh& ncmesh, const int* part = NULL);
```

Comment:
For consistency with the remaining variables, can we rename part to partitioning and use nullptr?

Reply:
Resolved in https://github.com/mfem/mfem/pull/4190/commits/3e5e18797ca8427697547b8f9d1bc46815176ce9#diff-f34a87a4063ff4ad77110d8e062b2fe3fabd7598d40dbc5f39d8b9c81d1eb0ff.